### DISCUSSION TITLE
Differences in POD of GRACE Fo (GROOPS run in Uni Luxembourg vs. TU Graz)

### DISCUSSION THREAD

**Discussion by ParisaShafiei** [groops-devs/groops#95] [Q&A] [answered]
Dear developers, 

I have used the example scenario for doing the precise orbit determination for GRACE Follow On for the year 2020. For the comparison, I have used the final orbits you have provided on the TUG webpage "https://ftp.tugraz.at/outgoing/ITSG/" and have computed the residuals. 

Firstly, since both of the estimated orbits, i.e. my and TUG results, are determined based on the same theory, I am wondering why there are such big differences between these orbits. It should be noted that I have not changed any settings in the example senario. So, did you use different settings for GRACE Follow On - C when you were doing precise orbit determination? Or besides, the iterative processing mentioned in on the latest article at TUG, is there anything else that you are considering in getting the final orbits?

Secondly, I have noticed that there are two points missed in the estimated kinematic orbits in three components, but in the TUG estimated orbits they do not exist. So, my question is do you have any filtering program after doing the POD for the LEO satellites to fill the gaps? 

For the sake of comparison, I have uploaded some of the figures that I got by plotting the kinematic orbits and residuals for one day, i.e. 01 January 2022. 

Like always, thank you in advance for your cooperation and support and I am really looking forward to hearing from you as soon as possible. 

With Kind Regards, 
Parisa 

![X_KIN_Orbits_GRACEFoC_2020-01-01](https://user-images.githubusercontent.com/99748448/183924867-afd3757e-0cca-4417-b107-59835d9b8b28.png)

![X_KIN_Orbits_GRACEFoC_TUG_2020-01-01](https://user-images.githubusercontent.com/99748448/183924878-c063983b-e731-498f-bb30-35526744db51.png)

![X_Residuals_in_GROOPS_UNI LU_vs _TUG](https://user-images.githubusercontent.com/99748448/183924930-b192b08c-44ea-44f5-b6ce-b9137c65d039.png)

**Comment by bsuesser-rechberger** [comment]
Dear Parisa,

The example uses the repro3 GNSS products, which includes the ANTEX R3 transmitter ACVs from IGS. For our published orbits, however, we estimate the transmitter ACV itself. (described in Suesser-Rechberger et al. (2022). Improved precise kinematic LEO orbits based on the raw observation approach. Advances in Space Research, 69(10), 3559-3570. https://doi.org/10.1016/j.asr.2022.03.014)

Since we are constantly developing the software, it is also possible that other observations are now discarded.
Sometimes epochs are removed because some observations are not correct. And if the GPS orbits are slightly different, then epochs can remain or be discarded.

Best regards,
Barbara

**Comment by ParisaShafiei** [comment]
Dear Barbara, 

Thanks for your reply. As a matter of fact, I went through this article as well as the flowchart explaining the processing strategy of determining kinematic orbit determination of LEO satellites. Therefore, when I started processing, I was anticipating that my results based on the example scenario would be different with yours. However, those differences in cm level for all the three components. Now, it's so obvious how following the processing strategy, e.g. determining the transmitter's ACVs and using the corrected patterns, can affect the accuracy of the estimated orbits. And I have understood why there are some gaps inside the estimated orbits. 

Besides, I have also another question, for doing the validation by using the final orbits from AIUB, at first, I have converted the GROOPS kinematic orbits from CRF to SP3. Nevertheless, as you can see in the plots, it seems there is a periodicity in all the three components and I am wondering if it can be caused because of the way I have converted the orbits from CRF to SP3 or not. Indeed, I have used the program "orbit2sp3format" for doing the conversion and I have attached the screenshot of the settings used in this program. I would be plased if you tell me if the way I have converted these orbits to SP3 format is correct or not. If it is correct, did you see this periodicity in validating your results? 
 
Thank you again for the time you spend on answering my question. Your consideration is really appreciated

With Kind Regards, 
Parisa 

![X_Residuals_in_SP3_GROOPS_TUG_vs _AIUB](https://user-images.githubusercontent.com/99748448/184300773-783fbee7-84fb-4fd1-b863-6bb4aeb21fde.png)

![Y_Residuals_in_SP3_GROOPS_TUG_vs _AIUB](https://user-images.githubusercontent.com/99748448/184300819-cf9ef332-2907-4c13-b316-792cc867205c.png)
![Z_Residuals_in_SP3_GROOPS_TUG_vs _AIUB](https://user-images.githubusercontent.com/99748448/184300845-3b6b4ddc-3d69-4c46-a729-945342f4cbef.png)

![image](https://user-images.githubusercontent.com/99748448/184300952-5cb98da8-9c3f-4e41-9c75-f286df994404.png)

**Comment by tmayerguerr** [comment, marked as answer]
Dear Parisa,
The differences are as expected. The periodics are once per revolution and might be due to the remaining systematic errors in the orbit computation (errors in the GPS orbits and clocks, antenna center variations, ...)
regards
Torsten

**Reply by ParisaShafiei** [reply]
Dear Prof. Mayer, 

Thank you very much.

Therefore, you mean the way I converted the CRF orbits to SP3 is correct. However, since I have used the final TUG orbits from your website to do the validation, I am still now sure why the differences are in cm level, while is mentioned in the PhD thesis of Dr. Zehentner, the differences between TUG and AIUB are in mm level, I was anticipating to see the residuals in mm level and not in cm. 

Besides, for the periodic you mentioned they are once per revolution, is it also possible to see them when we plot the kinematic orbits for one day, e.g. 01-Jan-2020?

The time you spend on answering my question is really appreciated. 

With Kind Regards, 
Parisa
